starcheck/calc_ccd_temps.py

```diff
@@ -520,7 +520,7 @@ def make_check_plots(outdir, states, times, temps, tstart, tstop, char):
     logger.info('Making temperature check plots')
     for fig_id, msid in enumerate(('aca',)):
         temp_ymax = max(char.aca_t_ccd_planning_limit, np.max(temps))
-        temp_ymin = min(char.aca_t_ccd_penalty_limit, np.min(temps))
+        temp_ymin = min(char.aca_t_ccd_planning_limit - 1, np.min(temps))
```

Comment:
We could also just auto scale the plot minimum to the data, but I figured there would be no harm in explicitly setting the default min to max range as 1 degree .